content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue.md

```diff
@@ -76,6 +76,12 @@ Once you have enabled the "Require merge queue" setting, you can also access the
 -
   **Merge limits**: Select the minimum and maximum number of pull requests to merge in a single group (between `1` and `100`), and a timeout after which the queue should stop waiting for more entries and merge with fewer than the minimum number of pull requests. Exactly how many PRs are included in a group depends on a merge queue's settings:
 
+{% note %}
+
+**Note:** Merge limits do not combine `merge_group` **builds**. Merge limits only affect merges to the base branch once one or more `merge_group` has satisfied build checks.
+
+{% endnote %}
+
```

Comment:
See previous comment. We think that by adjusting the wording above we can address the confusion and therefore this note is not necessary. Agree?

```suggestion
```

Reply:
I am fine removing a separate note, but believe there should still be explicit mention of builds vs merge as this distinction is still not clear.